tsconfig.json

```diff
@@ -19,5 +19,5 @@
     "noUnusedParameters": true,
     "noUnusedLocals": true
   },
-  "include": ["src"]
+  "include": ["src", "cypress"]
```

Comment:
Given we used `"noEmit": true`, confirmed this has no implications except for the sole purpose of type-checking